`make` fails due to git.code.sf.net requiring authentication for clones

```
∃!isisⒶsandstorm:(master=)/code/sandstorm ∴ make
==== downloading capnproto ====
git clone https://github.com/sandstorm-io/capnproto.git deps/capnproto
Cloning into 'deps/capnproto'...
remote: Counting objects: 13835, done.
remote: Total 13835 (delta 0), reused 0 (delta 0), pack-reused 13835
Receiving objects: 100% (13835/13835), 12.58 MiB | 3.78 MiB/s, done.
Resolving deltas: 100% (8937/8937), done.
Checking connectivity... done.
==== downloading ekam ====
git clone https://github.com/sandstorm-io/ekam.git deps/ekam
Cloning into 'deps/ekam'...
remote: Counting objects: 1717, done.
remote: Total 1717 (delta 0), reused 0 (delta 0), pack-reused 1717
Receiving objects: 100% (1717/1717), 1.77 MiB | 949.00 KiB/s, done.
Resolving deltas: 100% (1239/1239), done.
Checking connectivity... done.
==== downloading libseccomp ====
git clone git://git.code.sf.net/p/libseccomp/libseccomp deps/libseccomp
Cloning into 'deps/libseccomp'...
Username for 'https://git.code.sf.net': 
Password for 'https://git.code.sf.net': 
fatal: Authentication failed for 'https://git.code.sf.net/p/libseccomp/libseccomp/'
Makefile:109: recipe for target 'deps/libseccomp' failed
make: *** [deps/libseccomp] Error 128
```

--- a/Makefile
+++ b/Makefile
@@ -88,6 +88,14 @@ test: sandstorm-$(BUILD)-fast.tar.xz
 # ====================================================================
 # Dependencies
 
+# We list remotes so that if projects move hosts, we can pull from their new
+# canonical location.
+REMOTE_capnproto=https://github.com/sandstorm-io/capnproto.git
+REMOTE_ekam=https://github.com/sandstorm-io/ekam.git
+REMOTE_libseccomp=https://github.com/seccomp/libseccomp
+REMOTE_libsodium=https://github.com/jedisct1/libsodium.git
+REMOTE_node-capnp=https://github.com/kentonv/node-capnp.git
+
 deps: tmp/.deps
 
 tmp/.deps: deps/capnproto deps/ekam deps/libseccomp deps/libsodium deps/node-capnp
@@ -97,33 +105,33 @@ tmp/.deps: deps/capnproto deps/ekam deps/libseccomp deps/libsodium deps/node-cap
 deps/capnproto:
 	@$(call color,downloading capnproto)
 	@mkdir -p deps
-	git clone https://github.com/sandstorm-io/capnproto.git deps/capnproto
+	git clone $(REMOTE_capnproto) deps/capnproto
 
 deps/ekam:
 	@$(call color,downloading ekam)
 	@mkdir -p deps
-	git clone https://github.com/sandstorm-io/ekam.git deps/ekam
+	git clone $(REMOTE_ekam) deps/ekam
 	@ln -s .. deps/ekam/deps
 
 deps/libseccomp:
 	@$(call color,downloading libseccomp)
 	@mkdir -p deps
-	git clone git://git.code.sf.net/p/libseccomp/libseccomp deps/libseccomp
+	git clone $(REMOTE_libseccomp) deps/libseccomp
 
 deps/libsodium:
 	@$(call color,downloading libsodium)
 	@mkdir -p deps
-	git clone https://github.com/jedisct1/libsodium.git deps/libsodium
+	git clone $(REMOTE_libsodium) deps/libsodium
 
 deps/node-capnp:
 	@$(call color,downloading node-capnp)
 	@mkdir -p deps
-	git clone https://github.com/kentonv/node-capnp.git deps/node-capnp
+	git clone $(REMOTE_node-capnp) deps/node-capnp
 
 update-deps:
 	@$(call color,updating all dependencies)
 	@(for DEP in capnproto ekam libseccomp libsodium node-capnp; do cd deps/$$DEP; \
-	    echo "pulling $$DEP..."; git pull; cd ../..; done)
+	    echo "pulling $$DEP..."; git pull $(REMOTE_$(DEP)) ; cd ../..; done)
 
 # ====================================================================
 # Ekam bootstrap and C++ binaries

